### PR TITLE
Maven incremental Build

### PR DESCRIPTION
**Pull request by desmax74**
Changes to serialize the classes when exchanged between two different classloader

### CODE REVIEW

File: drools-compiler/src/main/java/org/drools/compiler/kie/builder/impl/AbstractKieModule.java [hunk 1]
```diff
@@ -95,9 +95,9 @@
 
     private final Map<String, Results> resultsCache = new HashMap<String, Results>();
 
-    protected final ReleaseId releaseId;
+    protected  ReleaseId releaseId;
 
-    private final KieModuleModel kModuleModel;
+    private  KieModuleModel kModuleModel;
```
Comment (mariofusco) [inline drools-compiler/src/main/java/org/drools/compiler/kie/builder/impl/AbstractKieModule.java]: @desmax74 If you want to do this I'm afraid you'll have to also make the KieModuleModel (and all other classes and interfaces related to it) also Serializable.

**Comment by mariofusco** [comment]
ok to test

**Comment by mariofusco** [comment]
Invalid

**Comment by desmax74** [comment]
Outdated

**Pull request closed without merging**